vehicle/src/Vehicle/Verify/Specification/IO.hs

```diff
@@ -354,28 +374,34 @@ verifyProperty verifierSettings verificationCache address = do
   let propertyPlanFile = propertyPlanFileName verificationCache address
   PropertyVerificationPlan {..} <- readPropertyVerificationPlan propertyPlanFile
 
+  let json = jsonOutput verifierSettings
+  -- Determine progressBar only for human mode
   result <- case queryMetaData of
-    Trivial status -> return $ PropertyCompleted (Trivial status)
+    Trivial status ->
+      return $ PropertyCompleted (Trivial status)
     NonTrivial structure -> do
-      logCompilerSection MinDetail ("Verifying property" <+> quotePretty address) $ do
-        -- Perform the verification
-        let numberOfQueries = propertySize queryMetaData
-        progressBar <- createPropertyProgressBar address numberOfQueries
-        let readerState = (verifierSettings, verificationCache, progressBar)
-        (errorOrResult, Sum numberOfQueriesExecuted) <-
-          runWriterT (runExceptT (runReaderT (verifyPropertyBooleanStructure structure) readerState))
-
-        -- The progress bar is only closed when all queries are run.
-        -- Not all queries are run, (e.g. short-circuited counter-example found or error occured).
-        -- In this case we have to close it manually.
-        when (numberOfQueriesExecuted < numberOfQueries) $ do
-          closePropertyProgressBar progressBar
-
-        case errorOrResult of
-          Left err -> return $ PropertyErrored err
-          Right result -> return $ PropertyCompleted $ NonTrivial result
-
-  outputPropertyResult verifierSettings verificationCache address result
+      -- Log property header
+      logCompilerSection MinDetail ("Verifying property" <+> quotePretty address) (return ())
```

Comment:
This change needs to reverted. You need the `do` at the end, and the stuff underneath to be indented as it was before, because this prints stuff both before and after what happens in the middle.